<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIGNUPVIEW</class>
 <widget class="QDialog" name="SIGNUPVIEW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>29</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请设置用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passWord">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>79</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请设置密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rePsd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>179</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密保问题：您的电话号码是</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SIGN_UP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
